### ISSUE TITLE
Windows OCO Agents not reporting to Server

### ISSUE THREAD

**Issue by KSJakobsen** [schorschii/OCO-Server#38]
Hi,

we are currently testing your OCO and at first glance it looks like a really promissing Tool.

We setup the OCO Server on a Debian 12. And the Server works fine. But we do have a problem with the Agents. It seems like not all of the Agents are reporting to the Server.

![Screenshot 2023-11-30 131459](https://github.com/schorschii/OCO-Server/assets/44235327/bcb0f50a-82d8-4085-a77e-f85f308106e0)

Could you advice what could be the problem, or are there any logs to find out what is going wrong ?

**Comment by schorschii** [comment]
Yes, please start the OCO agent in a terminal and post the output here. More information: [agent troubleshooting](https://github.com/schorschii/oco-agent#troubleshootingdebugging).

**Comment by schorschii** [comment]
Thank, this is a bug. Bitlocker itself is not the issue here (I'm using the agent myself on computers with Bitlocker). The command `bcdedit` seems to output non-ASCII chars on these computers. I already know how to fix it since it appeared on another place previously.

Just out of interest, can you please post the output of `bcdedit` from one of these computers?

**Comment by KSJakobsen** [comment]
bcdedit

Windows-Start-Manager
---------------------
Bezeichner              {bootmgr}
device                  partition=\Device\HarddiskVolume1
path                    \EFI\Microsoft\Boot\bootmgfw.efi
description             Windows Boot Manager
locale                  de-DE
inherit                 {globalsettings}
fverecoverymessage      Sie haben keinen Wiederherstellungsschlüssel? Wenden Sie sich an den IT-Helpdesk oder gehen Sie zu Ihrem Self Service Portal: https://sophos.com/ssp
default                 {current}
resumeobject            {92b196d5-0fb5-11ed-bca7-0c37966c1f4e}
displayorder            {current}
toolsdisplayorder       {memdiag}
timeout                 30

Windows-Startladeprogramm
-------------------------
Bezeichner              {current}
device                  partition=C:
path                    \windows\system32\winload.efi
description             Windows 11
locale                  de-DE
inherit                 {bootloadersettings}
recoverysequence        {e675deb5-0fbc-11ed-8b97-0c37966c1f4e}
displaymessageoverride  Recovery
recoveryenabled         Yes
isolatedcontext         Yes
allowedinmemorysettings 0x15000075
osdevice                partition=C:
systemroot              \windows
resumeobject            {92b196d5-0fb5-11ed-bca7-0c37966c1f4e}
nx                      OptIn
bootmenupolicy          Standard
hypervisorlaunchtype    Auto

**Comment by schorschii** [comment]
The issue here is the `ü` in your custom recovery message :D
A nice edge case! I'll fix this and release a new version in the next few days.

**Comment by schorschii** [comment]
Please test again with [v1.0.3](https://github.com/schorschii/OCO-Agent/releases/tag/v1.0.3).

**Comment by KSJakobsen** [comment]
Hi,

works perfectly thanks.
